main.go

```diff
@@ -17,6 +17,14 @@ var (
 	leafCertPath = flag.String("leafCertPath", "/etc/server/cert-chain.pem", "The leaf certificate to use for serving TLS")
 	leafKeyPath  = flag.String("leafKeyPath", "/etc/server/key.pem", "The private key of the leaf certificate to serve TLS with")
 
+	peerTrustDomainName = flag.String("peerTrustDomain", "spiffe://cluster-2", "The trust domain name to federate with")
+	peerEndpointAddress = flag.String("peerEndpointAddress", "35.193.205.112", "The address of the remote trust domain's bundle endpoint")
```

Comment:
Let's use a dedicated IP "240.0.0.10" for now. In the cluster, we create a service entry with a virtual IP for the external IP. We can use DNS name, but that needs to configure CoreDNS, I just don't want to worry about it for now :)